pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "pybind11~=2.6.1", "numpy", "scipy", "matplotlib"]
```

Comment:
@marcinwrobel1986 does this require mean that pybind11 is automatically pulled ? 
because we need a specific pybind11 that i forked and is a submodule of pressiodemoapps. is that going to be a problem?

Reply:
@fnrizzi this are requirements by build system. These are pulled automatically when building a package. I took this pybind11 version from pybind docs. It means it will take latest version, but not older than 2.6.1. Actually we build the dynamic library (.so) with cmake with the `build.py` script

Reply:
this won't work, because weed to use a special forked version of pybind11